main.py

```diff
@@ -283,6 +283,10 @@ def log_bad_request(request):
     except AttributeError:
         logging.debug("Bad request:{0}".format(str(request)))
 
+@app.route("/test")
```

Comment:
It is not necessary, of course. I used just to ensure that all the endpoints function correctly. You want me to remove it and re-submit the pull request?